src/mats_l1_processing/photometer.py

```diff
@@ -0,0 +1,93 @@
+#%%
+import numpy as np
+from mats_l1_processing.instrument import Photometer
+from pandas import DataFrame
+
+def calibrate_pm(df: DataFrame, photometer: Photometer):
+    
+# ===============================================================================================================
+# Calibration starts here
+# PM1 is Bkg photometer, 754BP3 nm / PM2 is A-band photometer, 763BP8 nm
+#   Change to bits/s
+    PM1_Sig_bit = df['PM1S']/df['PM1SCNTR']
+    PM1_Tpd_bit = df['PM1A']/df['PM1ACNTR']
+    PM1_Tif_bit = df['PM1B']/df['PM1BCNTR']
+    PM2_Sig_bit = df['PM2S']/df['PM2SCNTR']
+    PM2_Tpd_bit = df['PM2A']/df['PM2ACNTR']
+    PM2_Tif_bit = df['PM2B']/df['PM2BCNTR']
+    pm_texp = round(df['PM1SCNTR']/166) # exposure time in seconds, 1 second "integration" is 166-167 sampling cycles
+
+# ===========================================================
+# Calibrate data
+
+#   extracting variables from dict
+    bitar = photometer.cal_therm['bitar'] # 0.1 to 4095 bits in steps of 0.1 bit, array of float64 (1,40950)
```

Comment:
Most people working on this code probably know Swedish, and calling it `bitar` is a little bit cute, but consider if this is an appropriate name for it. ;)

I don't suggest changing it, since evidently this is what it is called in `Photometer`, so possibly more hassle than it is worth.

Same goes for `Temperatur`.

Reply:
yes, I would need to open the mat files or manually rename them here, too much of an hassle :D (also I guess its understandable for an english speaker even if it is a bit strange spelling ;D )